backend/api/Controllers/InspectionFindingsController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers
+{
+    [ApiController]
+    [Route("inspection-findings")]
+    public class InspectionFindingsController : ControllerBase
+    {
+        private readonly IInspectionService _inspectionService;
+        private readonly ILogger<InspectionFindingsController> _logger;
+        public InspectionFindingsController(
+            ILogger<InspectionFindingsController> logger,
+            IInspectionService inspectionService
+        )
+        {
+            _logger = logger;
+            _inspectionService = inspectionService;
+
+        }
+
+        /// <summary>
+        /// Add a new inspection finding - need isarStepId to run this
+        /// </summary>
+        /// <remarks>
+        /// <para> This query adds a new area to the database </para>
+        /// </remarks>
+        [HttpPost("add-findings")]
+        [Authorize(Roles = Role.Admin)]
+        [ProducesResponseType(typeof(AreaResponse), StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status409Conflict)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult<InspectionFindings>> AddFindings([FromBody] InspectionFindingsQuery inspectionFinding)
+        {
+            _logger.LogInformation("Updating inspection findings for inspection with isarStepId '{inspectionFinding.IsarStepId}'", inspectionFinding.IsarStepId);
```

Comment:
```suggestion
            _logger.LogInformation("Updating inspection findings for inspection with isarStepId '{Id}'", inspectionFinding.IsarStepId);
```

Reply:
I believe this is addressed now, hence resolving.

Reply:
Maybe not, why are we doing this ?

Reply:
We are doing it for readability. There is not need to specify the entire thing within the string as it is automatically associated with the parameter you put afterwards. Therefore just `Id` or `IsarStepId` is a more suitable name.

Reply:
Fixed now. Hence resolving.